helm-charts/hbase-chart/templates/meta/_zkscript.tpl

```diff
@@ -17,8 +17,25 @@ function shutdown() {
   echo "Stopping Zookeeper"
   $HBASE_HOME/bin/hbase-daemon.sh stop zookeeper
   if [ $exit_status != 0 ]; then
-	  echo "Leader stopped, sleeping for 60 seconds"
-	  sleep 60
+    IFS=$'\n' read -d '' -ra ZKs <<< $($HBASE_HOME/bin/hbase zkcli config 2> /dev/null  | grep participant | grep -o '=.*:2888' | cut -d : -f 1 | cut -c 2-)
```

Comment:
you can't hardcode ports. they can be configured differently

Reply:
addressed